### DISCUSSION TITLE
Choosing batches

### DISCUSSION THREAD

**Discussion by tomerlevy1095** [brianhie/scanorama#148] [Ideas]
Hello,

While using scanorama, I saw that it is only possible to use 1 batch key. In some cases, several columns in the data could be considered as batch. I was wondering if it is possible to implement the option to use several batch keys with Scanorama.

**Comment by brianhie** [comment]
Hi @tomerlevywis, even though there's only one key, you can collapse multiple batches into a single field like "key1value_key2value" etc.

**Reply by tomerlevy1095** [reply]
Thank you!
